tests/licensedcode/data/slic-tests/identification/repo

```diff
@@ -123,7 +123,10 @@ if sys.version_info[0] == 3:
   import urllib.error
 else:
   import imp
-  import urllib2
+  try:
```

Comment:
This is a test data file that is NOT part of the code we run, just something we test license detection on... so this file should not be modified at all.